src/main/resources/schema.sql

```diff
@@ -1,6 +1,24 @@
+CREATE TABLE IF NOT EXISTS people (
+    id BIGINT generated always as identity PRIMARY KEY,
```

Comment:
Postgres (or maybe SQL standard... not sure xD) has a nice built-in for generate / sequences. You can define it as `id BIGSERIAL PRIMARY KEY`

That combines the sequence and BIGINT and does all the work for you under the hood